### PR TITLE
docs(refactor): tutorials MTA-1810

### PR DESCRIPTION
**Pull request by bene2k1**

### CODE REVIEW

File: tutorials/setup-minecraft/index.mdx [hunk 1]
```diff
@@ -6,67 +6,53 @@ content:
...
   posted: 2019-11-15
 ---
 
+
 ## Minecraft - Overview
 
 In 2011 one of the most popular video games was released: [Minecraft](https://www.minecraft.net/en-us/).
 
-It was developed by the Swedish company [Mojang](https://www.mojang.com/) and has been sold more than 176 million times. Today Minecraft counts more than 112 million monthly active players.
+Developed by the Swedish company [Mojang](https://www.mojang.com/), it has been sold more than 176 million times. Today Minecraft counts more than 112 million monthly active players.
```
Comment (RoRoJ) [inline tutorials/setup-minecraft/index.mdx:22]:
```suggestion
Developed by the Swedish company [Mojang](https://www.mojang.com/), it has reached over 176 million sales. Today Minecraft counts more than 112 million monthly active players.
```

File: tutorials/deploy-awstats/index.mdx [hunk 2]
```diff
@@ -60,37 +44,37 @@ Before installing AwStats server packages, make sure Apache2 HTTP server is inst
 
 By default, AwStats package is available in the Ubuntu repository.
 
-1. Install AwStats package
+1. Install AwStats package.
```
Comment (ldecarvalho-doc) [inline tutorials/deploy-awstats/index.mdx:47]:
```suggestion
1. Install the AwStats package.
```

File: storage/object/api-cli/import-copy.mdx [hunk 3]
```diff
@@ -32,7 +32,7 @@ When applied on a versioned bucket, the sync command copies only the most recent
 
 <Message type="requirement">
 
-  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)  
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)  
 	- You have [configured your API keys](/console/my-project/how-to/generate-api-key/)
```
Comment (bene2k1) [inline storage/object/api-cli/import-copy.mdx:36]:
```suggestion
- You have [configured your API keys](/console/my-project/how-to/generate-api-key/)
```

File: tutorials/collecting-visualizing-logs-elastic-stack/index.mdx [hunk 4]
```diff
@@ -41,9 +41,9 @@ The ELK stack consists of:
 
 <Message type="requirement">
 
-  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
-  - You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key)
-  - You have created [an Instance](/compute/instances/how-to/create-an-instance/) or a [Elastic Metal server](/compute/elastic-metal/how-to/create-server/) with at least 4GB of RAM
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [configured your SSH key](/console/my-project/how-to/create-ssh-key)
+-  You have created [an Instance](/compute/instances/how-to/create-an-instance/) or a [Elastic Metal server](/compute/elastic-metal/how-to/create-server/) with at least 4GB of RAM
```
Comment (bene2k1) [inline tutorials/collecting-visualizing-logs-elastic-stack/index.mdx:46]:
```suggestion
  -  You have created [an Instance](/compute/instances/how-to/create-an-instance/) or a [Elastic Metal server](/compute/elastic-metal/how-to/create-server/) with at least 4GB of RAM
```

File: tutorials/install-postgresql/index.mdx [hunk 5]
```diff
@@ -6,27 +6,11 @@ content:
...
-    url: https://www.scaleway.com/en/docs/installation-of-postgresql/#-Connecting-to-a-Postgres-Prompt-with-the-new-Role
-  - step: Creating and Querying Databases
-    type: HowToStep
-    url: https://www.scaleway.com/en/docs/installation-of-postgresql/#-Creating-and-Querying-Databases
 categories:
   - storage
   - compute
 dates:
-  validation: 2021-05-10
+  validation: 2021-03-01‡
```
Comment (ldecarvalho-doc) [inline tutorials/install-postgresql/index.mdx:13]:
```suggestion
  validation: 2021-03-01
```

File: tutorials/host-gitlab-runner/index.mdx [hunk 6]
```diff
@@ -32,9 +32,9 @@ In this tutorial, you will learn how to set up a GitHub Actions runner on a Scal
 	
 <Message type="requirement">
 
-  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
-  - You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key)
-  - You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu 20.04
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [configured your SSH key](/console/my-project/how-to/create-ssh-key)
+-  You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu 20.04
```
Comment (bene2k1) [inline tutorials/host-gitlab-runner/index.mdx:37]:
```suggestion
  -  You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu 20.04
```

File: tutorials/install-mattermost/index.mdx [hunk 7]
```diff
@@ -17,9 +17,9 @@ dates:
 
 <Message type="requirement">
 
-  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
-  - You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key)
-  - You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Bionic Beaver
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [configured your SSH key](/console/my-project/how-to/create-ssh-key)
+-  You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Bionic Beaver
```
Comment (bene2k1) [inline tutorials/install-mattermost/index.mdx:22]:
```suggestion
  -  You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Bionic Beaver
```

File: tutorials/install-openlitespeed-wordpress/index.mdx [hunk 8]
```diff
@@ -19,9 +19,9 @@ The application provides a web-based administration interface, compatibility wit
 
 <Message type="requirement">
 
-  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
-  - You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key)
-  - You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Bionic Beaver
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [configured your SSH key](/console/my-project/how-to/create-ssh-key)
+-  You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Bionic Beaver
```
Comment (bene2k1) [inline tutorials/install-openlitespeed-wordpress/index.mdx:24]:
```suggestion
  -  You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Bionic Beaver
```

File: tutorials/install-parse-server/index.mdx [hunk 9]
```diff
@@ -18,9 +18,9 @@ Parse provides a cloud-based backend service to build data-driven mobile applica
 
 <Message type="requirement">
 
-  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
-  - You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key)
-  - You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Focal Fossa
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [configured your SSH key](/console/my-project/how-to/create-ssh-key)
+-  You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Focal Fossa
```
Comment (bene2k1) [inline tutorials/install-parse-server/index.mdx:23]:
```suggestion
  -  You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Focal Fossa
```

File: tutorials/install-php-composer/index.mdx [hunk 10]
```diff
@@ -18,9 +18,9 @@ PHP [Composer](https://getcomposer.org/) is a package dependency management tool
 
 <Message type="requirement">
 
-  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
-  - You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key)
-  - You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Focal Fossa
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [configured your SSH key](/console/my-project/how-to/create-ssh-key)
+-  You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Focal Fossa
```
Comment (bene2k1) [inline tutorials/install-php-composer/index.mdx:23]:
```suggestion
  -  You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Focal Fossa
```

File: tutorials/install-wireguard/index.mdx [hunk 11]
```diff
@@ -30,9 +30,9 @@ dates:
 
 <Message type="requirement">
 
-  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
-  - You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key)
-  - You have [created an Instance](/compute/instances/how-to/create-an-instance) configured with [local boot](/compute/instances/how-to/use-boot-modes/#how-to-use-local-boot) and running on a Linux kernel ≥ 3.10.
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [configured your SSH key](/console/my-project/how-to/create-ssh-key)
+-  You have [created an Instance](/compute/instances/how-to/create-an-instance) configured with [local boot](/compute/instances/how-to/use-boot-modes/#how-to-use-local-boot) and running on a Linux kernel ≥ 3.10.
```
Comment (bene2k1) [inline tutorials/install-wireguard/index.mdx:35]:
```suggestion
  -  You have [created an Instance](/compute/instances/how-to/create-an-instance) configured with [local boot](/compute/instances/how-to/use-boot-modes/#how-to-use-local-boot) and running on a Linux kernel ≥ 3.10.
```

File: tutorials/lb-firewall-haproxy-pfsense/index.mdx [hunk 12]
```diff
@@ -24,9 +24,9 @@ This tutorial guides you through the installation of pfSense on a [Scaleway Dedi
 
 <Message type="requirement">
 
-  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
-  - You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key)
-  - You have [created an Elastic Metal server](/compute/elastic-metal/how-to/create-server) or have a [Scaleway Dedibox](https://www.scaleway.com/en/dedibox/)
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [configured your SSH key](/console/my-project/how-to/create-ssh-key)
+-  You have [created an Elastic Metal server](/compute/elastic-metal/how-to/create-server) or have a [Scaleway Dedibox](https://www.scaleway.com/en/dedibox/)
```
Comment (bene2k1) [inline tutorials/lb-firewall-haproxy-pfsense/index.mdx:29]:
```suggestion
  -  You have [created an Elastic Metal server](/compute/elastic-metal/how-to/create-server) or have a [Scaleway Dedibox](https://www.scaleway.com/en/dedibox/)
```

File: tutorials/librenms-monitoring/index.mdx [hunk 13]
```diff
@@ -21,9 +21,9 @@ The software is based on [PHP](https://php.net) and [MySQL (MariaDB)](/tutorials
 
 <Message type="requirement">
 
-  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
-  - You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key)
-  - You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Focal Fossa Beaver
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [configured your SSH key](/console/my-project/how-to/create-ssh-key)
+-  You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Focal Fossa Beaver
```
Comment (bene2k1) [inline tutorials/librenms-monitoring/index.mdx:26]:
```suggestion
  -  You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Focal Fossa Beaver
```

File: tutorials/load-overflow-proxmox-and-elastic-metal/index.mdx [hunk 14]
```diff
@@ -26,9 +26,9 @@ In this tutorial we are going to create a [Proxmox VE](https://www.proxmox.com/e
 
 <Message type="requirement">
 
-  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
-  - You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key)
-  - You have [two Elastic Metal servers](/compute/instances/how-to/create-an-instance) that run on Proxmox
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [configured your SSH key](/console/my-project/how-to/create-ssh-key)
+-  You have [two Elastic Metal servers](/compute/instances/how-to/create-an-instance) that run on Proxmox
```
Comment (bene2k1) [inline tutorials/load-overflow-proxmox-and-elastic-metal/index.mdx:31]:
```suggestion
  -  You have [two Elastic Metal servers](/compute/instances/how-to/create-an-instance) that run on Proxmox
```

File: tutorials/mailtrain-ubuntu-bionic/index.mdx [hunk 15]
```diff
@@ -17,9 +17,9 @@ dates:
 
 <Message type="requirement">
 
-  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
-  - You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key)
-  - You have [created an Instance](/compute/instances/how-to/create-an-instance/) running Ubuntu Bionic or later
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [configured your SSH key](/console/my-project/how-to/create-ssh-key)
+-  You have [created an Instance](/compute/instances/how-to/create-an-instance/) running Ubuntu Bionic or later
```
Comment (bene2k1) [inline tutorials/mailtrain-ubuntu-bionic/index.mdx:22]:
```suggestion
  -  You have [created an Instance](/compute/instances/how-to/create-an-instance/) running Ubuntu Bionic or later
```

File: tutorials/mariadb-ubuntu-bionic/index.mdx [hunk 16]
```diff
@@ -28,9 +28,9 @@ Even the command line tools are similar to `mysqldump` and `mysqladmin` still ha
 
 <Message type="requirement">
 
-  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
-  - You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key)
-  - You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Bionic Beaver
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [configured your SSH key](/console/my-project/how-to/create-ssh-key)
+-  You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Bionic Beaver
```
Comment (bene2k1) [inline tutorials/mariadb-ubuntu-bionic/index.mdx:33]:
```suggestion
  -  You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Bionic Beaver
```

File: tutorials/setup-mongodb-on-ubuntu/index.mdx [hunk 17]
```diff
@@ -0,0 +1,428 @@
...
+  ssh root@your.instance.ip.address
+  ```
+
+  If you do not know your server IP, you can list your existing servers using the [Scaleway CLI](https://github.com/scaleway/scaleway-cli#scaleway-cli-v2) `scw instance server list`.
+
+   <Message type="tip">
+
+    If you use the root user, you can remove the `sudo` before each command.
+
+  </Message>
```
Comment (ldecarvalho-doc) [inline tutorials/setup-mongodb-on-ubuntu/index.mdx:53]: the indentation seems a bit off

File: tutorials/mastodon-community/index.mdx [hunk 18]
```diff
@@ -44,9 +44,9 @@ Mastodon provides the possibility of using [S3 compatible Object Storage](/stora
 
 <Message type="requirement">
 
-  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
-  - You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key)
-  - You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Focal Fossa
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [configured your SSH key](/console/my-project/how-to/create-ssh-key)
+-  You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Focal Fossa
```
Comment (bene2k1) [inline tutorials/mastodon-community/index.mdx:49]:
```suggestion
  -  You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Focal Fossa
```

File: tutorials/matomo-analytics/index.mdx [hunk 19]
```diff
@@ -20,9 +20,9 @@ The tool is written in PHP and stores its data in a MySQL/MariaDB database.
 
 <Message type="requirement">
 
-  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
-  - You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key)
-  - You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Focal Fossa (20.04)
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [configured your SSH key](/console/my-project/how-to/create-ssh-key)
+-  You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Focal Fossa (20.04)
```
Comment (bene2k1) [inline tutorials/matomo-analytics/index.mdx:25]:
```suggestion
  -  You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Focal Fossa (20.04)
```

File: tutorials/netdata-monitoring/index.mdx [hunk 20]
```diff
@@ -19,9 +19,9 @@ The application can detect hundreds of metrics automatically, simplifying their
 
 <Message type="requirement">
 
-  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
-  - You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key)
-  - You have [created four Instances](/compute/instances/how-to/create-an-instance)
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [configured your SSH key](/console/my-project/how-to/create-ssh-key)
+-  You have [created four Instances](/compute/instances/how-to/create-an-instance)
```
Comment (bene2k1) [inline tutorials/netdata-monitoring/index.mdx:24]:
```suggestion
  -  You have [created four Instances](/compute/instances/how-to/create-an-instance)
```

File: tutorials/setup-mongodb-on-ubuntu/index.mdx [hunk 21]
```diff
@@ -0,0 +1,428 @@
...
+
+  ```
+  admin> exit
+  ```
+
+### Enabling authentication
+
+In order to enforce authentication, we need to enable authentication and restart the MongoDB daemon.
+
+1. Open the configuration file
```
Comment (ldecarvalho-doc) [inline tutorials/setup-mongodb-on-ubuntu/index.mdx:180]:
```suggestion
1. Open the configuration file.
```

File: tutorials/openvpn-instant-app/index.mdx [hunk 22]
```diff
@@ -24,9 +24,9 @@ OpenVPN InstantApp comes with:
 
 <Message type="requirement">
 
-  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
-  - You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key)
-  - You have installed a VPN client on your local machine
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [configured your SSH key](/console/my-project/how-to/create-ssh-key)
+-  You have installed a VPN client on your local machine
```
Comment (bene2k1) [inline tutorials/openvpn-instant-app/index.mdx:29]:
```suggestion
  -  You have installed a VPN client on your local machine
```

File: tutorials/setup-mongodb-on-ubuntu/index.mdx [hunk 23]
```diff
@@ -0,0 +1,428 @@
...
+        Docs: https://docs.mongodb.org/manual
+    Main PID: 21449 (mongod)
+      Memory: 153.2M
+      CGroup: /system.slice/mongod.service
+              └─21449 /usr/bin/mongod --config /etc/mongod.conf
+  ```
+
+  Press `q` to exit.
+
+5. Ensure that the daemon restarts automatically at boot
```
Comment (ldecarvalho-doc) [inline tutorials/setup-mongodb-on-ubuntu/index.mdx:217]:
```suggestion
5. Ensure that the daemon restarts automatically at boot.
```

File: tutorials/setup-mongodb-on-ubuntu/index.mdx [hunk 24]
```diff
@@ -0,0 +1,428 @@
...
+  We are connected to the `test` database.
+
+2. Test that the access is restricted with the `show dbs` command:
+
+  ```
+  test> show dbs
+  MongoServerError: command listDatabases requires authentication
+  ```
+
+3. Exit the shell to proceed
```
Comment (ldecarvalho-doc) [inline tutorials/setup-mongodb-on-ubuntu/index.mdx:256]:
```suggestion
3. Exit the shell to proceed.
```

File: tutorials/setup-mongodb-on-ubuntu/index.mdx [hunk 25]
```diff
@@ -0,0 +1,428 @@
...
+
+  </Message> 
+
+  ```
+  net:
+    port: 27017
+    bindIp: 127.0.0.1,IP_of_MongoHost
+  ```
+
+3. Restart the daemon
```
Comment (ldecarvalho-doc) [inline tutorials/setup-mongodb-on-ubuntu/index.mdx:373]:
```suggestion
3. Restart the daemon.
```

File: tutorials/setup-mongodb-on-ubuntu/index.mdx [hunk 26]
```diff
@@ -0,0 +1,428 @@
...
+
+  ```
+  systemctl restart mongod
+  ```
+
+4. Check the daemon status
```
Comment (ldecarvalho-doc) [inline tutorials/setup-mongodb-on-ubuntu/index.mdx:379]:
```suggestion
4. Check the daemon status.
```

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**